packages/orbit-components/src/Seat/__examples__/LABEL.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import * as React from "react";
+
+import Seat from "../index";
+
+export default {
+  Example: () => <Seat label="MZ" />,
+  info: {
+    title: "Seat with label",
+    description: "Seat can have a label text inside",
+  },
+};
```

Comment:
New examples should be imported in `examples.js` to be visible in the Storybook.